stack-9.0.2.yaml

```diff
@@ -0,0 +1 @@
+resolver: lts-19.31
```

Comment:
Let's use latest stackage snapshot per each ghc version (see "Latest releases per GHC version" on [stackage.org](https://www.stackage.org/))

For 9.0.2 it's lts-19.33, in other config you're using latest already.